### PR TITLE
Feature/count header

### PR DESCRIPTION
**Pull request by sd109**
Adds an extra HTTP header to all successful responses with a count of the number of elements operated on while calculating the result of reduction. 

This probably makes the `count` endpoint obsolete and worth removing. I can remove that endpoint as part of this PR and bump semver to `0.2.0` if that's the route we want to take.

### CODE REVIEW

**Comment by sd109** [comment]
I guess we'll also need to update the compliance test suite so that CI continues to pass if we decide to remove the count endpoint.

**Comment by markgoddard** [comment]
I suppose the count endpoint might be useful if you only want the count. Otherwise you'd need to request something else and get the count from the header.

File: src/operations.rs [hunk 1]
```diff
@@ -13,6 +13,8 @@ use ndarray_stats::{errors::MinMaxError, QuantileExt};
 // Bring trait into scope to use as_bytes method.
 use zerocopy::AsBytes;
 
+// TODO: Remove count method from API once count-as-response-header is implemented
```
Comment (markgoddard) [inline src/operations.rs:16]: Remove if keeping count

**Review by markgoddard — COMMENTED** [review]
Nice work

**Comment by markgoddard** [comment]
Please also update the list of response headers in the README.

**Comment by markgoddard** [comment]
Need to also consider how to test this using compliance tests, and more generally how to handle divergence of the API over time.

**Review by markgoddard — APPROVED** [review]

**Pull request merged**